backend/src/main/java/asegroup1/api/daos/location/LocationDaoImpl.java

```diff
@@ -1,21 +1,52 @@
 package asegroup1.api.daos.location;
 
 
-import asegroup1.api.daos.DaoImpl;
-import asegroup1.api.models.LocationData;
-import org.springframework.stereotype.Repository;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.stream.Collectors;
 
 import javax.transaction.Transactional;
 
+import org.springframework.stereotype.Repository;
+
+import asegroup1.api.daos.DaoImpl;
+import asegroup1.api.models.LocationData;
+
 /**
- * @author Greg Mitten
- * gregoryamitten@gmail.com
+ * @author Greg Mitten gregoryamitten@gmail.com
+ * @author Rikkey Paal
  */
 
 @Transactional
 @Repository
 public class LocationDaoImpl extends DaoImpl<LocationData> {
+
+	private static final String TABLE_NAME = "location_data";
+
 	public LocationDaoImpl() {
 		setCurrentClass(LocationData.class);
 	}
+
+	@SuppressWarnings("unchecked")
+	public List<LocationData> getUserLocations(String userID) {
+		return (List<LocationData>) getEntityManager()
```

Comment:
Should assign `getEntityManager()` as a local variable and then open and close the transaction.